src/components/Button/Button.tsx

```diff
@@ -68,7 +68,7 @@ export interface Props {
   /**
    * Available _stylistic_ variations available for the Button component
    */
-  variant?: 'primary' | 'bare' | 'link';
+  variant?: 'primary' | 'bare' | 'link' | 'table-header';
```

Comment:
Dunno why the 'table-header' variant is missing. I remove it in the next PR, but at this point in the stack, it's still being used here and in another file.